resources/assets/less/bem/landing-logo.less

```diff
@@ -28,25 +28,15 @@
  	margin: auto;
  	z-index: @z-index--nav-float;
 
- 	&__header {
-        // Visually hidden h1 for SEO purposes
-        border: 0;
-        clip: rect(0 0 0 0);
-        height: 1px;
-        margin: -1px;
-        overflow: hidden;
-        padding: 0;
-        position: absolute;
-        width: 1px;
-    }
-
-    &__overlay,
-    &__glow,
-    &__timing,
-    &__bounce {
-    	position: absolute;
-    	width: 100%;
-    	height: 100%;
+    &__pulse {

```

Comment:
See [this](https://github.com/ppy/osu-web/blob/nanaya/new-header/resources/assets/less/bem/logo.less).